### PR TITLE
Bug/2819 forbidden chars in batch query and not in uri param type pattern

### PR DESCRIPTION
**Pull request by kzangeli**
Fixes issue #2819.
Also, instead of checking for forbidden chars for Entity::isPattern, now a simple check for "true" or "false" is done. Not sure what to do about this CNR wise. I'm pretty sure there is no real change for the users. The broker is simply a tiny little bit faster like this.

### CODE REVIEW

File: src/lib/apiTypesV2/Entity.cpp [hunk 1]
```diff
@@ -199,10 +199,24 @@ std::string Entity::check(ApiVersion apiVersion, RequestType requestType)
     return std::string(errorMsg);
   }
 
-  if (forbiddenIdChars(apiVersion, id.c_str()))
+  if (isPattern == "")
```
Comment (fgalan) [inline src/lib/apiTypesV2/Entity.cpp]: Although we have an issue for an overall fix for this (https://github.com/telefonicaid/fiware-orion/issues/2791) let's try to start using `empty()` in new/modified code from now on.

Comment (kzangeli) [inline src/lib/apiTypesV2/Entity.cpp]: Not gonna be easy to change the chip ...
But, sure, let's try at least :-)

Comment (kzangeli) [inline src/lib/apiTypesV2/Entity.cpp]: Fixed in 8baaa28

**Comment by fgalan** [comment]
LGTM

**Pull request merged**